templates/hosted-engine.conf.j2

```diff
@@ -25,6 +25,15 @@ ca_subject="C=EN, L=Test, O=Test, CN=Test"
 vdsm_use_ssl=true
 gateway={{ he_gateway }}
 bridge={{ he_mgmt_network }}
+{% if he_network_test is defined %}
+network_test={{ he_network_test }}
+{% endif %}
+{% if he_tcp_t_address is defined %}
```

Comment:
We can change it to 
{% if he_tcp_t_address is not none %}
initializing he_tcp_t_address to none in defaults/main.yml; please also add this to README.md

Reply:
If he_tcp_* is initialized in defaults/main.yml , it is more consistent to skip this {if} and to write
`tcp_t_address=`
`tcp_t_port=`
like done in the iSCSI values.

Reply:
Yes, I agree